res/controllers/Hercules-DJ-Console-RMX-hid-scripts.js

```diff
@@ -336,7 +336,7 @@ RMX.define_hid_format = function() {
   c.add_control(pid, "filterHighKill",    "[Channel1]", "button", 2, 0x20);
   c.add_control(pid, "filterMidKill",     "[Channel1]", "button", 2, 0x40);
   c.add_control(pid, "filterLowKill",     "[Channel1]", "button", 2, 0x80);
-  c.add_control(pid, "pitch_reset",       "[Channel1]", "button", 3, 0x01);
+    c.add_control(pid, "pitch_set_default",       "[Channel1]", "button", 3, 0x01);
```

Comment:
I think the whitespace has changed too, can you make that consistent?

Reply:
Will do - this was pre-commit, not me

Reply:
Done

Reply:
pre-commit insists on the whitespaces (ignore) but it also wants the `_unusedVar` format.
Do you want to fix that or just carry on and merge?

Reply:
Let's just merge it!